src/frontend/src/app/pages/profile-edit/anonymous-profile/anonymous-profile.component.ts

```diff
@@ -14,10 +17,13 @@ import { AnonymousProfile } from '~core/state';
           <h2 class="mb-8">Welcome to the CodeGov organisation</h2>
           <p>
             If you would like to become a reviewer,
-            <a href="/apply">apply now</a>.
+            <a [href]="applyLink">apply now</a>.
           </p>
           <p>When requested, provide this ID to a CodeGov admin:</p>
-          <p class="mb-8 font-bold">{{ userProfile.id }}</p>
+          <div class="mb-8 flex flex-row items-center">
```

Comment:
We should look at this layout on mobile, maybe we could see how it looks if we use `flex-col md:flex-row` or something similar.